app/drivers/usb.c

```diff
@@ -251,6 +250,8 @@ static uint8_t desc_device_qualifier[] =
 	0x00,
 };
 
+static char usb_serialnumber_string[13];
```

Comment:
Why not in `self`?

Reply:
Lol, there were so many separate static variables that I missed that we had already created a `me`.

Reply:
You mean a `self`, there's no `me` here, not sure what you're talking about :P

Reply:
lol. I am not sure where this concept of `me` comes from either, never heard about it. Anyway, I moved it to `self` and seems to work.